cpp/proto/arcticc/pb2/azure_storage.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package arcticc.pb2.azure_storage_pb2;
+
+message Config {
+    string container_name = 1;
+    string credential_name = 2;
+    string credential_key = 3;
+    string endpoint = 4;
+    uint32 max_connections = 5;
+    uint32 request_timeout = 6;
+    string prefix = 7;
+    bool https = 8;
+    bool connect_to_azurite = 9;
```

Comment:
Can't the `endpoint` take care of it? If not, it's a pretty poorly designed mock library....

Reply:
I made this comment with the assumption that there should not be any Azurite-specific behaviour for it to be a meaningful mock...